dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -66,6 +66,139 @@ data:
             "x": 0,
             "y": 3
           },
+          "id": 35,
+          "panels": [],
+          "title": "Application Deletion SLO",
+          "type": "row"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+          },
+          "description": "The objective for the given time window.\n",
+          "gridPos": {
+            "h": 7,
+            "w": 10,
+            "x": 0,
+            "y": 4
+          },
+          "id": 36,
+          "links": [],
+          "options": {
+            "code": {
+              "language": "plaintext",
+              "showLineNumbers": false,
+              "showMiniMap": false
+            },
+            "content": "<h1><center>99% of application deletions must succeed</center></h1>",
+            "mode": "html"
+          },
+          "pluginVersion": "9.3.8",
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "${datasource}"
+              },
+              "refId": "A"
+            }
+          ],
+          "title": "SLO",
+          "type": "text"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PDCCF087A30F0737B"
```

Comment:
Is it suppose to be explicit `uid`? 
I see that we used a variable in other places (uid": "${datasource}")

Reply:
D'oh, not sure what happened there, must've accidentally switched it in my testing. It should be good now